data/model/user_timeline_event.py

```diff
@@ -88,3 +88,11 @@ class APITimelineEvent(BaseModel):
     user_name: constr(min_length=1)
     created: NonNegativeInt
     metadata: APIEventMetadata
+    hidden: Optional[bool]
+
+class HideUserTimelineEvent(BaseModel):
+    id: NonNegativeInt
+    user_id: NonNegativeInt
+    event_type: UserTimelineEventType
+    event_id: NonNegativeInt
+    created: Optional[datetime]
```

Comment:
Why optional, this column is NOT NULL in table schema?

Reply:
Sorry my bad, I had put it there during testing, and forgot to remove it 😅